.github/workflows/pythonpackage.yml

```
@@ -9,6 +9,10 @@ on:
       - master
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Just use `{{ github.ref }}` so that any job from a new commit cancels all jobs from older ones, as suggested in https://github.com/boutproject/boutdata/pull/51#discussion_r677990146
```suggestion
  group: ${{ github.ref }}
```